dm/run.py

```diff
@@ -363,15 +363,15 @@ def remove_ortho_tiles():
         progressbc.send_update(30)
         
         
-        #filter TiePoints (better distribution, avoid clogging)
-        #kwargs_schnaps = {
-        #    'ext': image_ext,
-        #    'mm3d': mm3d
-        #    
-        #}
-	#    system.run('{mm3d} Schnaps .*.{ext} MoveBadImgs=1'.format(**kwargs_schnaps))
-        #
-        #progressbc.send_update(5)
+        filter TiePoints (better distribution, avoid clogging)
```

Comment:
we can try this but in my experience schnaps has some issues. also, we want to stick with RadialStd as it is more flexible for calibration.. i'll run some tests and we like were you are going with these enhancements

Reply:
The problem with Schnaps is that , need to load whole Homol folder in RAM